ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx

```diff
@@ -53,13 +54,16 @@ const Header = ({
             imageUrl={imageUrl}
             imageUrlTemplate={imageUrlTemplate}
             imageWidth={imageWidth}
+            hideImage={isMediaOpen}
           />
         ) : null}
         <div
           css={[
-            isHeaderImage && styles.textContainerWithImage,
-            !isHeaderImage && styles.textContainerWithoutImage,
-            isMediaOpen && styles.textContainerMediaOpen,
+            isWithImageLayout && styles.textContainerWithImage,
+            !isHeaderImage &&
+              !mediaCollections &&
```

Comment:
Does this work? (it would be more readable)

```suggestion
            !isWithImageLayout
```